src/NNR/ctl/albedo_clim.ctl

```diff
@@ -0,0 +1,13 @@
+DSET /nobackup/MODIS/051/Level3/ALBEDO/clim/albedo_555nm.clim.%d2%mc%y4.nc
```

Comment:
1. All these paths are on calculon? perhaps put a note somewhere for the default. It may be already in the howto?

Reply:
i think all of these control files will need to be reorganized in a future update.